phy/cluster/manual/default_settings.py

```diff
@@ -0,0 +1,40 @@
+
+# -----------------------------------------------------------------------------
+# Correlograms
+# -----------------------------------------------------------------------------
+
+# Number of time samples in a bin.
+manual_clustering.correlograms_binsize = 20
+
+# Number of bins (odd number).
+manual_clustering.correlograms_winsize_bins = 2 * 25 + 1
+
+# Maximum number of spikes for the correlograms.
+manual_clustering.correlograms_n_spikes_max = 20000000
```

Comment:
for some reason I thought we needed that many for Nick's dataset, but now it looks like it works just fine with 1M. I just updated it.